data/AlchemicalArtist.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Chris Denton",
+  "displayStatsPublic": false,
+  "type": "personal",
+  "bio": "artist, musician, human",
+  "avatar": "https://GitHub.com/alchemicalArtist.png",
+  "links": [
+    {
+      "name": "Subscribe on YouTube!",
+      "url": "https://YouTube.com/@AlchemicalArtist",
+      "icon": "faYoutube"
```

Comment:
The icons should have a capital `Fa`, so it would be `FaYoutube`, you can check the icon names here in the docs https://linkfree.eddiehub.io/icons